steemconnect4j/src/test/java/com/hapramp/steemconnect4j/tests/StringUtilsTest.java

```diff
@@ -150,6 +150,25 @@ public void testGetOperationsStringObject() {
     assertEquals(expected, StringUtils.getOperationsString(input));
   }
 
+  @Test
+  public void testStringFormat() {
+    String follower = "bxute";
+    String following = "unittestaccount";
+    String params = StringUtils.getCommanSeparatedObjectString(
+        RpcJsonUtil.getKeyValuePair("required_auths", "[]"),
+        RpcJsonUtil.getKeyValuePair("required_posting_auths", "[\"" + follower + "\"]"),
+        RpcJsonUtil.getKeyValuePair("id", "\"follow\""),
+        RpcJsonUtil.getKeyValuePair("json", "\"" + RpcJsonUtil.stringify(
+        StringUtils.getCommanSeparatedArrayString("\"follow\"",
+          StringUtils.getCommanSeparatedObjectString(
+            RpcJsonUtil.getKeyValuePair("follower", "\"" + follower + "\""),
+            RpcJsonUtil.getKeyValuePair("following", "\"" + following + "\""),
+            RpcJsonUtil.getKeyValuePair("what", "[\"blog\"]")))) + "\""
+      )
+    );
+    System.out.println(params);
```

Comment:
This should be an assertion or something. Simply logging the result won't help in testing it.